pkg/runtime/service_controller.go

```diff
@@ -153,11 +198,20 @@ func (c *serviceController) BindControllerManager(mgr ctrlrt.Manager, cfg ackcfg
 		c.reconcilers = append(c.reconcilers, rec)
 	}
 
-	rec := NewAdoptionReconciler(c, c.log, cfg, c.metrics, cache)
-	if err := rec.BindControllerManager(mgr); err != nil {
-		return err
+	adoptionInstalled, err := c.GetAdoptedResourceInstalled(mgr)
+	errLogger := c.log.WithName("adoption-setup")
```

Comment:
nit: 
```suggestion
	adoptionLogger := c.log.WithName("adoption-setup")
```

Reply:
You could also pass it to the adoption reconciler `rec := NewAdoptionReconciler(c, adoptionLogger, cfg, c.metrics, cache)`